project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+          - or:
+            - Infiltration|50 ACH50
+            - Infiltration|40 ACH50
+            - Infiltration|30 ACH50
+            - Infiltration|25 ACH50
+            - Infiltration|20 ACH50
+            - Infiltration|18.5 ACH50
+            - Infiltration|15 ACH50
+            - Infiltration|11.25 ACH50
+            - Infiltration|10 ACH50
+            - Infiltration|8 ACH50
+            - Infiltration|7.5 ACH50
+            - Infiltration|7 ACH50
+            - Infiltration|7 ACH50, 0.5 Shelter Coefficient
+            - Infiltration|6 ACH50
+            - Infiltration|5.25 ACH50
+            - Infiltration|5 ACH50
+            - Infiltration|4.5 ACH50
+            - Infiltration|4 ACH50
+            - Infiltration|3.75 ACH50
+        costs:
+          - value: 0.50 # To be decided
+                        # $0.50/ft2 is from LBNL data for general air sealing (https://doi.org/10.20357/B7FP4D).
+                        # Interpolated between 20% and 40% reductions to get $0.44/ft2,
+                        # then used 1.14 multiplier to inflate from Jan 2019 to Mar 2022
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      - &infiltration_IECC2021_add_ERV
+        option: Mechanical Ventilation|ERV, 72%
+        apply_logic:
+        - and:
+            - or:
+              - Mechanical Ventilation|None
+              - Mechanical Ventilation|Exhaust
+              - Mechanical Ventilation|Supply
+            - or:
+              - Infiltration|50 ACH50
+              - Infiltration|40 ACH50
+              - Infiltration|30 ACH50
+              - Infiltration|25 ACH50
+              - Infiltration|20 ACH50
+              - Infiltration|18.5 ACH50
+              - Infiltration|15 ACH50
+              - Infiltration|11.25 ACH50
+              - Infiltration|10 ACH50
+              - Infiltration|8 ACH50
+              - Infiltration|7.5 ACH50
+              - Infiltration|7 ACH50
+              - Infiltration|7 ACH50, 0.5 Shelter Coefficient
+              - Infiltration|6 ACH50
+              - Infiltration|5.25 ACH50
+              - Infiltration|5 ACH50
+              - Infiltration|4.5 ACH50
+              - Infiltration|4 ACH50
+              - Infiltration|3.75 ACH50
+        costs:
+          - value: 0.50 # To be decided
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60
+      - &roof_insulation_to_R30
+        option: Insulation Roof|Finished, R-30
+        apply_logic: 
+          - and:
+            - or:
+              - Geometry Attic Type|Finished Attic or Cathedral Ceilings
+              - Geometry Building Level MF|Top
+            - or:
+              - Insulation Roof|Finished, Uninsulated
+              - Insulation Roof|Finished, R-7
+              - Insulation Roof|Finished, R-13
```

Comment:
What about `- Insulation Roof|Finished, R-19`

Reply:
I refer the apply logic in the EUSS RR1, which applies to dwelling units with roof insulation of R-13 or less. Shall we change the logic to "applies to dwelling units with roof insulation less than R-30"?

Reply:
I don't have strong feelings either way on this. For EUSS RR1, this apply_logic meant no homes built after 2000 (and almost no homes built after 1990) were eligible, but we could add a separate criteria that only homes built before 2000 are eligible. I think that's more complexity than needed.

Conclusion: let's include R-19 as eligible here, and not worry about a vintage restriction (will only make QC harder).

Reply:
done